.github/workflows/ci.yml

```diff
@@ -95,3 +95,20 @@ jobs:
           config: .github/grcov.yml
       - name: Upload Results
         uses: codecov/codecov-action@v2
+
+  test-unit-all-features:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Generate lockfile
+        run: cargo generate-lockfile
+      - name: Setup Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-test-${{ hashFiles('**/Cargo.lock') }}
+      - run: cargo test --all-features
```

Comment:
Wouldn't it be more appropriate to measure the code coverage in this job?

Reply:
I think so, switching the flags as suggested :+1: